tests/Jira/Api/Client/AbstractClientTestCase.php

```diff
@@ -30,16 +30,27 @@ protected function setUp()
 		$this->client = $this->createClient();
 	}
 
-	public function testGetRequest()
+	/**
+	 * @dataProvider getRequestWithKnownHttpCode
+	 */
+	public function testGetRequestWithKnownHttpCode($http_code)
 	{
 		$data = array('param1' => 'value1', 'param2' => 'value2');
-		$trace_result = $this->traceRequest(Api::REQUEST_GET, $data);
+		$trace_result = $this->traceRequest(Api::REQUEST_GET, array_merge(['http_code' => $http_code], $data));
```

Comment:
The library is PHP 5.3+ compatible and PHP 5.4+ specific syntax (short arrays in this case) should not be used.